.github/workflows/container-image.yml

```
@@ -47,6 +47,31 @@ jobs:
       with:
         image: "samba-server:latest"
         retention_days: 1
+  build-server-centos:
+    runs-on: ubuntu-latest
+      #env:
+      #      BUILDAH_FORMAT: oci
+    steps:
+      -  uses: actions/checkout@v3
+      -   name: Build the CentOS based server image
+          run: make OS_NAME=centos build-server
+    - name: Upload server image
+      uses: ishworkh/docker-image-artifact-upload@v1
+      with:
+        image: "samba-server:centos-latest"
+        retention_days: 1
+
+  test-centos-server:
+    needs: build-centos-server
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Download server image
+      uses: ishworkh/docker-image-artifact-download@v1
+      with:
+        image: "samba-server:centos-latest"
+    - name: Test the server image
+      run: tests/test-samba-container.sh
```

We need to make use of `LOCAL_TAG` variable to consume _samba-server:centos-latest_.

```suggestion
      run: LOCAL_TAG=samba-server:centos-latest tests/test-samba-container.sh
```

---

> We need to make use of `LOCAL_TAG` variable to consume _samba-server:centos-latest_.

done, thanks! :-)